src/ipc/index.ts

```diff
@@ -4,6 +4,7 @@ export enum IpcRequest {
   UpdateBuildStatus = 'UPDATE_BUILD_STATUS',
   ChooseFolder = 'CHOOSE_FOLDER',
   SaveFile = 'SAVE_FILE',
+  LoadLogFile = 'LOAD_LOG_FILE',
```

Comment:
I think in this case using graphql API would be a more appropriate. We are using Electron IPC API only when it is required to interact with the operating system and/or application window. Everything else is graphql only.

Reply:
I have a question in this case. What is the source of the logs isn't it a local file?

Reply:
The vast majority of the logs are produced by the graphql api server.

Reply:
And where do we store them? I need more information from where should we fetch them. My understanding was to display the logs from the log file which is in the folder the button on this page opens.

Reply:
You are displaying the correct logs, but Jurg would like GraphQL used to communicate between the frontend and backend to retrieve the log data instead of an IPC request.  This application uses GraphQL to transfer data between the frontend and backend, a traditional client/server model using http requests, we want to remain consistent and not have some data transfer done through GraphQL and others through IPC.